sdk/communication/AzureCommunicationChat/Source/Signaling/TrouterSettings.swift

```diff
@@ -27,16 +27,17 @@
 import Foundation
 import TrouterClientIos
 
+// TODO: Test in Prod
 let defaultRegistrationData: TrouterUrlRegistrationData? = TrouterUrlRegistrationData.create(
-    withApplicationId: "AcsiOS",
+    withApplicationId: "AcsiOS_test",
     registrationId: nil,
     platform: "SPOOL",
     platformUiVersion: "0.0.0",
-    templateKey: "AcsiOS_Chat_1.1",
+    templateKey: "AcsiOS_Chat_test_1.3",
     productContext: nil,
     context: ""
 ) as? TrouterUrlRegistrationData
```

Comment:
This is still awaiting resolution?

Reply:
yes, still waiting for changes to be deployed in PROD